packages/connect-core/src/connections/IAppConnected.ts

```diff
@@ -0,0 +1 @@
+export default interface IAppConnected {}
```

Comment:
I removed both of them since app connectors are now extending `App`, but the idea was that `IAppConnected` would have been the wrapping object (e.g. [`Voting`](https://github.com/aragon/connect/blob/d7518138e48e8200801670db8f30ec671ea71e3d/packages/connect-thegraph-voting/src/entities/Voting.ts)), while `IAppConnector` would have been the internal connector class (e.g. [`VotingConnectorTheGraph`](https://github.com/aragon/connect/blob/d7518138e48e8200801670db8f30ec671ea71e3d/packages/connect-thegraph-voting/src/connector.ts)).